### PR TITLE
Profile view / Mongo Test

### PR DESCRIPTION
**Pull request by jay-khatri**
Added  profile view:
- Link from the results page to individual profiles
- Profile page currently displays all user data (including ID)
Added Mongo test:
- Basic functionality implemented
- Needs to be updated when function signatures change
- Note: all tests pass, but there are several deprecated functions that are inherited from "pymongo", we may need to address this

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,9 @@
+config.py
 .DS_Store
 .idea/
 th-secrets/
 venv/
+env/
```
Comment (him229) [inline .gitignore:6]: what are you using /env for?

File: app.py [hunk 2]
```diff
@@ -97,5 +97,13 @@ def upload():
         new_count = data["count"]
         return render_template('upload.html', count=count, sample_data=preview ,count_diff=new_count, msg="For uploading data from a request")
 
+
+@app.route('/profile/<person_id>')
+def profile(person_id):
+    unique_person = Person.objects.get(id=person_id)
+    print(unique_person.__dict__)
```
Comment (him229) [inline app.py]: remove the print statement?

File: templates/profile.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        <h2>Profile: {{ entry.first_name + " " + entry.last_name }}</h2>
+    </div>
+</div>
+<div class="container">
+    {% for attr in entry.__dict__['_fields_ordered'] %}
+        <p>{{ attr|string + ": " + entry[attr]|string }}</p>
+    {% endfor %}
+</div>
+</body>
+</html>
```
Comment (him229) [inline templates/profile.html]: just add a new line at the end

**Review by him229 — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (jay-khatri) [reply .gitignore:6]: I renamed my "venv" to "env" for whatever reason, I can go back.

File: app.py (hunk 2 above)
Comment (jay-khatri) [reply app.py]: done.

**Pull request merged**